airflow/providers/amazon/aws/transfers/snowflake_to_s3.py

```diff
@@ -0,0 +1,150 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""Transfers data from Snowflake into a S3 Bucket."""
+from typing import List, Optional
+
+from airflow.models import BaseOperator
+from airflow.providers.snowflake.hooks.snowflake import SnowflakeHook
+from airflow.utils.decorators import apply_defaults
+
+
+class SnowflakeToS3Operator(BaseOperator):
+    """
+    UNLOAD a query or a table from Snowflake to S3. It is not necessary a conn to S3 because
+    Snowflake handles it. You have to follow one of the setups in:
+    https://docs.snowflake.com/en/user-guide/data-load-s3-config.html
+
+    :param stage: Copy the data into a Snowflake Stage. This allows you to use the method
+        'Configuring AWS IAM User Credentials' for unloading data to s3. If this is true,
+        s3_bucket and file_format are not necessaries.
+    :type stage: bool
```

Comment:
```suggestion
    :param use_stage: Copy the data into a Snowflake Stage. This allows you to use the method
        'Configuring AWS IAM User Credentials' for unloading data to s3. If this is true,
        s3_bucket and file_format are not necessaries.
    :type use_stage: bool
```
WDYT? If yes, then obviously you have to change the references. (I do not comment on them. :))

Reply:
Actually this is a mistake, stage is a string. I have already changed it. Thanks!